src/components/CorporateCampaign/CampaignLoanRow.vue

```diff
@@ -0,0 +1,406 @@
+<template>
```

Comment:
This is a highly modified version of the category row used on the homepage which is not powered by a loan channel and mean to prototype the layout. Eventual goal is to allow 1 type of filter per row with a friendly header and selector for the target facet. ie. "Find loans in ##SomeCountry##". Additionally, it has a "load more" button to just keep adding loans to the row.